Activities/Scripting/UiPath.Scripting.Activities/Resources/ActivitiesMetadata.json

```diff
@@ -0,0 +1,26 @@
+{
+  "AssemblyRelativePath": "UiPath.Scripting.Activities.dll",
+  "Activities": [
+    {
+      "AssemblyQualifiedName": null,
+      "FullName": "UiPath.Scripting.Activities.InvokePowerShellCore",
```

Comment:
why was this file added? is there a requirement for studio web support? if so, then view models must be implemented as well (most likely will be the case, since XAML designers will no loger be used)